features/base/media/components/VideoTrack.js

```diff
@@ -29,12 +29,21 @@ class VideoTrack extends Component {
      */
     render() {
         const videoTrack = this.props.videoTrack;
+        let stream = null;
+
+        if (videoTrack) {
+            // XXX We may want not to start showing video until video stream
+            // has started to play anywhere else.
+            stream = this.props.waitForVideoStarted && !videoTrack.videoStarted
+                ? null
```

Comment:
If waitForVideoStarted is set to true, Video won't be rendered until ```videoTrack.videoStarted``` is truthy. In other case it will show video no matter if ```videoTrack.videoStarted``` is truthy or not.